src/server/verification/api/ZoomAPI.js

```diff
@@ -126,33 +115,56 @@ class ZoomAPI {
         return encodeURIComponent(parameterValue)
       }
 
+      log.debug('Calling Zoom API:', config)
```

Comment:
does config holds all request info? or we need something else?
remove facemap/audtitrail from logging